docs/USER_RECORD.md

```diff
@@ -557,6 +557,15 @@ automatically killed when the user logs out. This is enforced by
 [`systemd-logind.service`](https://www.freedesktop.org/software/systemd/man/systemd-logind.service.html). If
 false any processes left around when the user logs out are left running.
 
+`freezeSession` → A boolean. If true the user's session is frozen whenever the home
+directory is locked (i.e. whenever the contents of the user's home directory are made
+inaccessible until the user re-authenticates). If false the user's session will not be
+frozen whenever the home directory is locked, but the kernel may still freeze any task
+that tries to access files in the user's home directory. This can lead to edge-cases that
+may lead to data loss (for example: the display server starts killing apps frozen by the
+kernel because it sees them as unresponsive). Thus, we recommend that this setting is
+left unset or is set to true.
```

Comment:
why make this a part of the user record? if you want to disable this because of nvidia crap or so, then this seems like an inherently local/driver thing, i.e. the nvidia driver should just drop in a file that does Environment=… for systemd-suspend.service (and related units), but why would the user record configure this?

Reply:
Just as an alternative way to disable it, more user-facing for people that have their own custom suspend hooks that wait on user services. I imagined they have the same hooks copied onto every system.

I'd be happy to drop it